tool/appveyor.ps1

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2014, Google Inc. Please see the AUTHORS file for details.
+# All rights reserved. Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+param([string]$action)
+
+function throw_if_process_failed {
+    param([string]$message)
```

Comment:
Do you know, is a 2-space indent or 4-space indent common in `ps1` files? Most of the other files in the repo are using 2 space indent.